cli/install/install.go

```diff
@@ -644,10 +651,98 @@ func copyBuildedTarantool(binPath, incPath, binDir, includeDir, version string,
 	return err
 }
 
-// installTarantool installs selected version of tarantool.
-func installTarantool(version string, binDir string, incDir string, flags InstallationFlag,
+//go:embed Dockerfile.tnt.build
+var tarantoolBuildDockerfile []byte
+
+func installTarantoolInDocker(binDir string, incDir string, installCtx InstallCtx,
 	distfiles string) error {
-	versions, err := getTarantoolVersions(flags.Local, distfiles)
+	tmpDir, err := ioutil.TempDir("", "docker_build_ctx")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpDir)
+
+	currentUser, err := user.Current()
+	if err != nil {
+		return err
+	}
+
+	goTextEngine := engines.NewDefaultEngine()
+	dockerfileText, err := goTextEngine.RenderText(string(tarantoolBuildDockerfile),
+		map[string]string{"uid": currentUser.Uid})
+	if err != nil {
+		return err
+	}
+
+	// Write docker file (rw-rw-r-- permissions).
+	if err = ioutil.WriteFile(filepath.Join(tmpDir, "Dockerfile"), []byte(dockerfileText),
```

Comment:
Why not use a constant name for the docker file? it is already in use in two places.